src/CMakeLists.txt

```diff
@@ -563,6 +563,7 @@ set(TINYSPLINE_INSTALL_PKGCONFIG_DIR
 )
 
 option(TINYSPLINE_FLOAT_PRECISION "Build TinySpline with float precision." OFF)
+option(TINYSPLINE_SERIALIZATION "Build TinySpline with serialization support." ON)
```

Comment:
Please add this option to the [ci.yml](https://github.com/msteinbeck/tinyspline/blob/master/.github/workflows/ci.yml#L12) matrix so that this config is tested in the CI.